adoc-mode.el

```diff
@@ -46,6 +46,15 @@
 ;;; Variables:
 
 (require 'markup-faces) ; https://github.com/sensorflo/markup-faces
+
+(defface markup-native-code-face ;; TODO: This should go into `markup-faces.el'
+  '((((background light))
+     (:background "cornsilk" :extend t))
+    (((background dark))
+     (:background "saddlebrown" :extend t)))
+  "For code blocks that are highlighted natively."
```

Comment:
Btw, what is this face used for exactly? If something is font-locked natively it's supposed to use the faces of the original major mode, so I'm puzzled about its purpose.

Reply:
I adapted this from Orgmode and I like this feature very much. It gives code blocks a gray background. So it is easier to differentiate them from normal text.

Reply:
Got it.